app/src/main/res/drawable/edittext_background_focused.xml

```diff
@@ -2,7 +2,7 @@
 <!-- res/drawable/rounded_edittext_focused.xml -->
 <shape xmlns:android="http://schemas.android.com/apk/res/android"
     android:shape="rectangle" android:padding="10dp">
-    <solid android:color="#FFFFFF"/>
+    <solid android:color="@color/colorAccent"/>
```

Comment:
Please undo the changes to this file. This effects everything in the app and we already defined how EditTexts need to look.